certbot/plugins/manual.py

```diff
@@ -41,7 +45,16 @@ class Authenticator(common.Plugin):
 
     description = "Manually configure an HTTP server"
 
-    MESSAGE_TEMPLATE = """\
+    MESSAGE_TEMPLATE = {
+        "dns-01": """\
```

Comment:
Change this template to be:
```
To prove control of the domain {domain}, please deploy a DNS TXT record with the following value:

{validation}

Once this is deployed, 
```
(the next line will be `Press ENTER to continue`).  You'll need to supply `achall.domain` as a parameter to the template.